judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/TestStaticQueryDao.java

```diff
@@ -0,0 +1,206 @@
+            "REQ-MDL-001",
+            "REQ-MDL-002",
+            "REQ-MDL-003",
+            "REQ-TYPE-001",
+            "REQ-TYPE-002",
+            "REQ-TYPE-004",
+            "REQ-TYPE-005",
+            "REQ-TYPE-006",
+            "REQ-TYPE-007",
+            "REQ-TYPE-008",
+            "REQ-ENT-001",
+            "REQ-ENT-002",
+            "REQ-ENT-011"
+
+    })
+    void testPrimitiveQueriesDao() {
+
+        EntityWithPrimitives entityWithPrimitives = entityWithPrimitivesDao.create(EntityWithPrimitives.builder().build());
+
+        //Integer query
+        assertEquals(1, integerWithoutParamQueryDao.execute());
+        assertEquals(2, integerWithParamQueryDao.execute(IntegerWithParamQueryParameter.builder().withNum(1).build()));
+        assertEquals(2, integerWithDefaultParamQueryDao.execute(IntegerWithDefaultParamQueryParameter.builder().build()));
+
+        //Scaled query
+        assertEquals(2.34, scaledWithoutParamQueryDao.execute());
+        assertEquals(3.34, scaledWithParamQueryDao.execute(ScaledWithParamQueryParameter.builder().withNum(1.0).build()));
+        assertEquals(4.68, scaledWithDefaultParamQueryDao.execute(ScaledWithDefaultParamQueryParameter.builder().build()));
+
+        //String query
+        assertEquals("test", stringWithoutParamQueryDao.execute());
+        assertEquals("testhello", stringWithParamQueryDao.execute(StringWithParamQueryParameter.builder().withString("hello").build()));
+        assertEquals("testhello", stringWithDefaultParamQueryDao.execute(StringWithDefaultParamQueryParameter.builder().build()));
+
+        //Boolean query
+        assertEquals(true, booleanWithoutParamQueryDao.execute());
+        assertEquals(false, booleanWithParamQueryDao.execute(BooleanWithParamQueryParameter.builder().withBool(false).build()));
+        assertEquals(true, booleanWithDefaultParamQueryDao.execute(BooleanWithDefaultParamQueryParameter.builder().build()));
+
+        //Date query
+        assertEquals(LocalDate.of(2022, 7, 11), dateWithoutParamQueryDao.execute());
+        assertEquals(
+                LocalDate.of(2023, 4, 3),
+                dateWithParamQueryDao.execute(DateWithParamQueryParameter.builder().withDateYear(2023).withDateMonth(4).withDateDay(3).build())
+        );
+        assertEquals(
+                LocalDate.of(2023, 3, 3),
+                dateWithDefaultParamQueryDao.execute(DateWithDefaultParamQueryParameter.builder().build())
+        );
+
+        //Time
+        assertEquals(LocalTime.of(22, 59, 59), timeWithoutParamQueryDao.execute());
+        assertEquals(
+                LocalTime.of(12, 24, 45),
+                timeWithParamQueryDao.execute(TimeWithParamQueryParameter.builder().withTimeHour(12).withTimeMinute(24).withTimeSecond(45).build())
+        );
+        assertEquals(
+                LocalTime.of(13, 4, 5),
+                timeWithDefaultParamQueryDao.execute(TimeWithDefaultParamQueryParameter.builder().build())
+        );
+
```

Comment:
Timestamp is missing (adding it should be an "almost" copy-paste)

Reply:
[1a37435](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/175/commits/1a37435b20fb4d2843ff2f08ba668cdab23b636b)